CHANGES.rst

```diff
@@ -1198,6 +1198,9 @@ astropy.units
 - For ``Quantity.to_string()``, ensure that the precision argument is also
   used when the format is not latex. [#11145]
 
+- For CDS units and tables, recognize ``-`` as indicating dimensionless and
```

Comment:
@bsipocz - changelog entries are always so darn tricky. Would it have helped to include `---` here? Happy to make quick PR to change it.

Reply:
yes, it would have helped. I started with the changelog, then tried to grep for it in the diff before falling back to the heavy machinery of git bisect.